AggregateMetrics/AggregateMetrics.Tests/AzureWebApp/AzureWebAppTest.cs

```diff
@@ -22,5 +26,30 @@ public void TestPerformanceCounterValuesAreCorrectlyRetrievedUsingFlexiblePerfor
 
             Assert.IsTrue(metric.Value >= 0);
         }
+
+        [TestMethod]
+        public void TestGetDefaultCountersWorks()
+        {
+
+
+            List<MetricTelemetry> metrics = DefaultCounters.Instance.GetCountersHttp();
+            int countersFirstIteration = metrics.Count;
```

Comment:
what is this metrics? Is it private variable? Ideally tests should not share anything and be as atomic as possible